### ISSUE TITLE
Countermodel not entirely printing on ARM

### ISSUE THREAD

**Issue by philzook58** [draperlaboratory/cbat_tools#294]
Make a file called main.s
```
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@ 
@    Return 5
@
@ To compile this:
@
@    arm-linux-gnueabi-as -g main.asm -o main.o
@    arm-linux-gnueabi-ld -entry=main main.o -o main
@
@

@ ----------------------------------------------
    .text

@ ----------------------------------------------
@ The MAIN function
@ ----------------------------------------------
    .global main
    .type main, %function
main:
    mov r0, #5
    bx lr
    .size main, .-main
```
If you don't have arm toolchain installed
`apt install -y binutils-arm-linux-gnueabi gcc-arm-linux-gnueabi`

Compile to main.o with `arm-linux-gnueabi-as -g main.asm -o main.o`

Example run (on arm branch):

```

philip@philip-desktop:~/Documents/ocaml/VIBES/resources/exes/simple$ objdump -d main.o
main.o:     file format elf32-littlearm
Disassembly of section .text:
00000000 <main>:
   0:   e3a00005        mov     r0, #5
   4:   e12fff1e        bx      lr
philip@philip-desktop:~/Documents/ocaml/VIBES/resources/exes/simple$ bap wp --func=main main.o main.o --postcond="(assert (= R1_orig R1_mod))"
Evaluating precondition.
Checking precondition with Z3.
UNSAT!
philip@philip-desktop:~/Documents/ocaml/VIBES/resources/exes/simple$ bap wp --func=main main.o main.o --postcond="(assert (= R2_orig R2_mod))"
Evaluating precondition.
Checking precondition with Z3.
SAT!
Model:
        SP  |->  0x3f800082
        R9  |->  0x00000000
        R8  |->  0x00000000
        R7  |->  0x00000000
        R6  |->  0x00000000
        R5  |->  0x00000000
        R4  |->  0x00000000
        R3  |->  0x00000000
        R2  |->  0xffffffff
        R12 |->  0x00000000
        R11 |->  0x00000000
        R10 |->  0x00000000
        R1  |->  0x00000000
        R0  |->  0x00000000
        LR  |->  0x00000000
        mem_orig |-> [
                else |-> 0x00]
        mem_mod = mem_orig
```

**Comment by codyroux** [comment]
Originally we printed both models. It was confusing, since all values were the same. This might be more of a bug that the registers should *all* be equal.

**Comment by fortunac** [comment]
Fixed in #290
